ui/lib/core/addon/components/form-field.hbs

```diff
@@ -187,17 +187,37 @@
       @value={{or (get @model this.valuePath) @attr.options.defaultValue}}
       @onChange={{this.onChangeWithEvent}}
     />
+  {{else if (eq @attr.options.editType "toggleButton")}}
+    {{! Togglable Input }}
+    <Toggle
+      @name="toggle-{{@attr.name}}"
+      @onChange={{this.toggleButton}}
+      @checked={{this.toggleInputEnabled}}
+      data-test-toggle={{@attr.name}}
+      disabled={{this.disabled}}
+    >
+      <span class="has-text-weight-bold is-large">{{this.labelString}}</span><br />
+      <div class="description has-text-grey" data-test-toggle-subtext>
+        <span>
+          {{#if this.toggleInputEnabled}}
+            {{@attr.options.enabledSubText}}
```

Comment:
I noticed we already have`helperTextEnabled/helperTextDisabled` as an options subkey - do you think it's worth naming these the same to minimize supported attr options  (which are already, admittedly, are a mess 🍝 🤯 )